app/database/db_build.sql

```diff
@@ -0,0 +1,16 @@
+BEGIN;
+
+DROP TABLE IF EXISTS nuggets;
+
+CREATE TABLE nuggets(
+  id SERIAL PRIMARY KEY,
+  x_coordinate INTEGER NOT NULL,
+  y_coordinate INTEGER NOT NULL,
```

Comment:
maybe call these lat and long instead of x_coordinate and y_coordinate?

also they are not integers (they will be a float e.g. 30.843929)